src/java.base/share/classes/sun/security/provider/PolicyFile.java

```diff
@@ -608,6 +608,9 @@ public Void run() {
                 pe.add(new PropertyPermission
                                 ("java.specification.version",
                                     SecurityConstants.PROPERTY_READ_ACTION));
+                pe.add(new PropertyPermission
+                        ("java.specification.maintenance.version",
```

Comment:
the indentation is not in sync with the lines above and below, maybe you could fix this.